### ISSUE TITLE
When reproducing, I noticed the loss decreased and then increased sharply

### ISSUE THREAD

**Issue by forest520** [fixie-ai/ultravox#170]
When reproducing the experiment, I found that after the loss decreased, it showed a clear upward trend again

I reproduced on 8 * 32GB-V100,here are my script:
`
nohup torchrun --nnodes=1 --nproc_per_node=8 --rdzv_id=123 --rdzv_backend=c10d --rdzv_endpoint=localhost:29500 ultravox/ultravox/training/train.py --config_path=ultravox/training/configs/release_config.yaml > train.log 2>&1 &
` 

here a the tensorboard log :
![image](https://github.com/user-attachments/assets/4b1e9593-0e87-4be3-bc44-be7016d6be2f)

Due to memory constraints, I had to modify the batch size and max steps:
```
text_model: "/data2/ultravox_data/model/Meta-Llama-3.1-8B-Instruct"
audio_model: "openai/whisper-medium"
loss_config:
  loss_function: "KL_Divergence"

train_sets:
  - name: librispeech-clean-continuation
  - name: librispeech-other-continuation
  - name: commonvoice-en-continuation


val_sets:
  - name: covost2-en-de
  - name: covost2-zh-en
  - name: peoplespeech-clean-transcription

batch_size: 3
max_steps: 115200 # x8x24 = 2,764,800
```

**Comment by Gpwner** [comment]
when  it almost done ,it come to be this,
![image](https://github.com/user-attachments/assets/02576741-6fa2-430b-945c-ec82d9972658)
Is this as expected?

**Comment by zqhuang211** [comment]
We haven’t experienced the same issue. Typically, the training loss continues to decrease and eventually stabilizes. It’s possible that your batch size is too small. Try using a larger value for `grad_accum_steps`, such as 8 (default: 1).

**Comment by Gpwner** [comment]
thanks,i will give a try.i will track this by the discussion https://github.com/fixie-ai/ultravox/discussions/270

**Comment by zqhuang211** [comment]
Sounds good! I suggest starting with the continuation dataset first to establish a strong baseline. I’ll close this for now.
